README.md

```diff
@@ -1,27 +1,18 @@
 Bowling Challenge in Ruby
 =================
 
-* Feel free to use google, your notes, books, etc. but work on your own
-* If you refer to the solution of another coach or student, please put a link to that in your README
-* If you have a partial solution, **still check in a partial solution**
-* You must submit a pull request to this repo with your code by 9am Monday week
+* This is the weekend challenge for week 5 of Makers Academy.
```

Comment:
I like that you have commented here your potential plans for an interface and that you can run it in irb. It might also be nice to see any diagramming if you did that 🙂